context/app/static/js/components/detailPage/BulkDataTransfer/BulkDataTransfer.jsx

```diff
@@ -36,7 +36,9 @@ function BulkDataTransfer() {
   return (
     <FilesContext.Provider value={filesContext}>
       <DetailPageSection id="bulk-data-transfer" data-testid="bulk-data-transfer">
-        <SectionHeader>Bulk Data Transfer</SectionHeader>
+        <SectionHeader iconTooltipText="Information about how to bulk download all files related to this dataset.">
```

Comment:
Not sure if it is reflected in the design, but we should use `entity_type` here as you do in `Attribution`.